jboss_wildfly/datadog_checks/jboss_wildfly/data/conf.yaml.example

```diff
@@ -184,9 +184,9 @@ init_config:
 
 
 
-  # Undertow
-  # inspired from:
-  # https://github.com/nlighten/wildfly_exporter/blob/master/client/src/main/java/nl/nlighten/prometheus/wildfly/UndertowExports.java
+  ## Undertow
+  ## inspired from:
```

Comment:
These two lines are weird. What does it means?

Reply:
idk.. it was there, i just double commented it.. we could remove them if you want.